cli/command/stack/kubernetes/conversion_test.go

```diff
@@ -165,25 +167,7 @@ func makeKubeService(service, stack, uid string, serviceType apiv1.ServiceType,
 	}
 }
 
-func withMode(mode string, replicas uint64) func(*swarm.Service) {
-	return func(service *swarm.Service) {
-		switch mode {
-		case "global":
-			service.Spec.Mode = swarm.ServiceMode{
-				Global: &swarm.GlobalService{},
-			}
-		case "replicated":
-			service.Spec.Mode = swarm.ServiceMode{
-				Replicated: &swarm.ReplicatedService{Replicas: &replicas},
-			}
-			withStatus(0, replicas)
-		default:
-			service.Spec.Mode = swarm.ServiceMode{}
-			withStatus(0, 0)
-		}
-	}
-}
-
+// TODO convertToServices currently doesn't set swarm.EndpointSpec.Ports
```

Comment:
Added a TODO for this; looks like the conversion doesn't return the ports in the _spec_ for the service